File-Formats/Presentation/Working-with-PowerPoint-presentation.md

```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
+
+## First slide number
+
+Change the first slide number of PowerPoint presentation using **FirstSlideNumber** API. 
```

Comment:
Change the first slide number of the PowerPoint presentation using the **FirstSlideNumber** API.